oaknational/Oak-Web-Application
# chore: point to sandbox hubspot on dev [LESQ-1152]

## Description

Music year: 1978

- points to the new sandbox environment for dev hubspot forms 

## How to test

**note: hubspot seems to be marking submissions from preview domain as spam, if that happens you will see the sumission [here](https://app.hubspot.com/submissions-spam/47200262/) and you can click release to add it into submissions

1. go through the sign up and onboarding journey /teachers/programmes/biology-secondary-ks4-higher-aqa/units/stem-cells-and-differentiation/lessons/stem-cells-in-animals/downloads
2. Check [here](https://app.hubspot.com/submissions/47200262/form/cfd18abd-ccf8-40b7-9fe3-44b5e0509198/submissions?redirectUrl=https%3A%2F%2Fapp.hubspot.com%2Fforms%2F47200262%3Fquery%3Daccount) to see your onboarding form has been submitted correctly
3.  Download a resource
4. Check [here](https://app.hubspot.com/submissions/47200262/form/02b980e3-b5c1-455a-bb40-f14cfbbb7243/submissions?redirectUrl=https%3A%2F%2Fapp.hubspot.com%2Fforms%2F47200262%3Fquery%3Dteacher%2Bhub)  to see your download form has been submitted correctly
5. Go to the newsletter sign up on the home page and sign up for the newsletter
6. Check [here](https://app.hubspot.com/submissions/47200262/form/c3c339ae-756c-4861-8445-740ebff59369/submissions/?redirectUrl=https%3A%2F%2Fapp.hubspot.com%2Fforms%2F47200262%3Fquery%3Dmain%2Bemail)  that the form submitted correctly
